<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-EQA1VQ0" failures="1" tests="3" name="Error Validations Test" time="56.298" errors="0" timestamp="2022-11-30T23:24:06 IST">
  <testcase classname="AnupamaProjects.tests.ErrorValidations" name="LoginErrorValidation" time="0.642"/>
  <testcase classname="AnupamaProjects.tests.ErrorValidations" name="ProductErrorValidation" time="23.588">
    <skipped/>
  </testcase> <!-- ProductErrorValidation -->
  <testcase classname="AnupamaProjects.tests.ErrorValidations" name="ProductErrorValidation" time="13.685">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at AnupamaProjects.tests.ErrorValidations.ProductErrorValidation(ErrorValidations.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- ProductErrorValidation -->
</testsuite> <!-- Error Validations Test -->
